### DISCUSSION TITLE
github and project description

### DISCUSSION THREAD

**Discussion by Tobi-De** [dymmond/esmerald#7] [Potential Issue]
Hi, hope you are doing great.
It's just a detail but this is what I saw on my github feed:

` Highly scalable, performant, easy to learn, easy to code and for every application.`

I think you are missing `web framework` or `python web framework` somewhere in there. It wasn't obvious to me what this was about when I saw it in my feed, of course I came to see what it was but some might skip it just for that.
It's just a detail but I wanted to point it out anyway, otherwise good job on what you're doing here, it looks interesting.

**Comment by tarsil** [comment]
Great point @Tobi-De ! Will update soon

**Comment by tarsil** [comment]
Let me know how you will get along @Tobi-De . A new release is coming next week with some  updates too.

**Reply by Tobi-De** [reply]
> Let me know how you will get along @Tobi-De . A new release is coming next week with some updates too.

Glad to hear it, I will definitely try it.

**Comment by tarsil** [comment]
@Tobi-De I hope you enjoy Esmerald. Maybe I can lock this conversation?

**Reply by Tobi-De** [reply]
Sure go ahead
